includes/Block/Selected_Stories_Block.php

```diff
@@ -69,6 +76,15 @@ class Selected_Stories_Block extends Latest_Stories_Block {
 	 * @return void
 	 */
 	public function init() {
+		/**
+		 * Filters the maximum number of stories users can select
+		 *
+		 * @since
+		 *
+		 * @param int $max_num_of_stories
+		 */
+		$this->max_num_of_stories = apply_filters( 'web_stories_selected_stories_limit', 20 );
```

Comment:
Can we not use the pre-defined class variable `$max_num_of_stories` instead of passing the `20`?

Reply:
Yes, we can. The only reason I initialized it with 0 and set 20 here is to use a filter. So do you prefer like this,
```php
...
protected $max_num_of_stories = 20;
...
...
...
$this->max_num_of_stories = apply_filters( 'web_stories_selected_stories_limit', $this->max_num_of_stories );

```

Should I update my code using this change?

Reply:
yes,